### PR TITLE
feat: add support the `valgrind include` by triple

### LINKED ISSUE #101: Add `IAI_CALLGRIND_<triple>_VALGRIND_INCLUDE`
For cross compilation like `CARGO_TARGET_<triple>_RUNNER`.

Example:

```
IAI_CALLGRIND_X86_64_UNKNOWN_LINUX_GNU_VALGRIND_INCLUDE
```

### PR DESCRIPTION
**Pull request by qRoC**
Add `IAI_CALLGRIND_<triple>_VALGRIND_INCLUDE` as an alternative to `IAI_CALLGRIND_VALGRIND_INCLUDE`

Partial closes #101

### CODE REVIEW

File: iai-callgrind/build.rs [hunk 1]
```diff
@@ -39,9 +42,21 @@ mod imp {
         println!("cargo:rustc-cfg=client_requests_support=\"{value}\"");
     }
 
-    fn build_native() {
+    fn include_dirs(target: &Target) -> impl Iterator<Item = String> {
+        [
+            Cow::Owned(format!(
+                "IAI_CALLGRIND_{}_VALGRIND_INCLUDE",
+                target.triple.replace('-', "_")
```
Comment (gamma0987) [inline iai-callgrind/build.rs:49]: I'd prefer the environment variable in upper case.

**Review by gamma0987 — CHANGES_REQUESTED** [review]
Thanks. Looks great.

File: iai-callgrind/build.rs (hunk 1 above)
Comment (qRoC) [reply iai-callgrind/build.rs:49]: trying to edit this evening

**Comment by gamma0987** [comment]
I'm going to finish this pr now.

**Pull request closed without merging**